Applications/LogisticRegression/jni/main.cpp

```diff
@@ -217,7 +217,8 @@ int main(int argc, char *argv[]) {
 
       try {
         float answer =
-          NN.forwarding({MAKE_SHARED_TENSOR(nntrainer::Tensor({o}))})[0]
+          NN.forwarding({MAKE_SHARED_TENSOR(nntrainer::Tensor({o}))},
+                        {MAKE_SHARED_TENSOR(nntrainer::Tensor({l}))})[0]
```

Comment:
I wonder what effect does this change has. 
IMO the only difference this will make is calculate loss, which in this case should not affect the running or the result.

Reply:
I guess it is temporal. we need to provide inference API. not call forward directly in the application. If we call directly.. there is error because hidden is not set properly.

Reply:
Got it.